frontend/src/component/context/ContextList/ContextList.tsx

```diff
@@ -73,6 +88,30 @@ const ContextList: VFC = () => {
                 ),
                 sortType: 'alphanumeric',
             },
+            ...(uiConfig.flags.segmentContextFieldUsage
+                ? [
+                      {
+                          Header: 'Used in',
+                          width: '60%',
+                          Cell: ({ row: { original } }: any) => (
+                              <TextCell
+                                  sx={{
+                                      color:
```

Comment:
This logic should be touched by some unit test

Reply:
Added